FunctionSignatureMap.php

```diff
@@ -9943,6 +9943,7 @@
 'simplexml_load_file' => ['SimpleXMLElement|false', 'filename'=>'string', 'class_name='=>'string', 'options='=>'int', 'ns='=>'string', 'is_prefix='=>'bool'],
 'simplexml_load_string' => ['SimpleXMLElement|false', 'data'=>'string', 'class_name='=>'string', 'options='=>'int', 'ns='=>'string', 'is_prefix='=>'bool'],
 'SimpleXMLElement::__construct' => ['void', 'data'=>'string', 'options='=>'int', 'data_is_url='=>'bool', 'ns='=>'string', 'is_prefix='=>'bool'],
+'SimpleXMLElement::__get' => ['SimpleXMLElement', 'name'=>'string'],
```

Comment:
This implements magic getters internally, but not through __get(). DynamicPropertyMap is what Phan uses to keep track of this type of edge case.

```
php > $x = new SimpleXMLElement('<b>x</b>');
php > $x->__get('b');

Warning: Uncaught Error: Call to undefined method SimpleXMLElement::__get() in php shell code:1
Stack trace:
#0 {main}
  thrown in php shell code on line 1
php > var_export($x->b);
SimpleXMLElement::__set_state(array(
))
```

The rest of the changes look good

Reply:
Just removed that line